.trunk/trunk.yaml

```diff
@@ -7,6 +7,8 @@ cli:
 repo:
     trunk_remote_hint: github.com/eslint/eslint
 
+telemetry: off
```

Comment:
Just checking if this change was intentional, as it was committed in https://github.com/eslint/eslint/pull/19675/commits/f64e8407baab2bd3d0550cbf7906fb5fce38d46d, which is titled "Move getRuleNumericSeverity to static" but doesn't actually move getRuleNumericSeverity to static.

Reply:
Hmm, good question. I don't know what happened to the rest of the changes.